core/arch/arm/plat-stm32mp1/drivers/stm32mp1_clk.c

```diff
@@ -213,46 +215,53 @@ struct stm32mp1_clk_pll {
 	enum stm32mp_osc_id refclk[REFCLK_SIZE];
 };
 
+#define N_S	0	/* Non-secure can access RCC interface */
+#define SEC	1	/* RCC[TZEN] protects RCC interface */
```

Comment:
Minor: how about `SEC` and `NSEC`?

Reply:
no strong opinion but I think `N_S`/`SEC` better differs versus `NSEC`/`SEC`.

Reply:
Fine with me